en/Community/workingCooperatively.md

```diff
@@ -4,7 +4,7 @@
 
 ## Getting Things Done
 
-Regardless of whether or not you've participated in large software development projects, if you're new to open source projects you may be in for some surprises. The leadership of such projects is very "flat" -- meaning that there isn't a lot of bureaucracy to deal with. On the other hand, you'll find that as a developer, you're given great freedom in finding interesting work and designing what gets built. The more code and ideas you contribute, the more you'll become a leader in the project.
+Regardless of whether or not you've participated in large software development projects, if you're new to open source projects you may be in for some surprises. The leadership of such (open source) projects is very "flat" -- meaning that there isn't a lot of bureaucracy to deal with. On the other hand, you'll find that as a developer, you're given great freedom in finding interesting work and designing what gets built. The more code and ideas you contribute, the more you'll become a leader in the project.
```

Comment:
the reason is , in that paragraph, u previously talk about "large software development projects", and "open source projects". 
hence we need to clarify wat exactly wc projects we are talking about

Reply:
I do not see value added by this change because the context is already set for open source projects by this phrase "if you're new to open source projects".

Reply:
ya true, though Howard Archer(The Reporter) had raised it , in his complaits